keps/sig-network/1435-mixed-protocol-lb/README.md

```diff
@@ -500,7 +514,7 @@ Graduating to GA means, that the feature flag checking is removed from the code.
 
 #### Beta -> GA Graduation
 
-TBD
+- All major cloud providers support this or indicate non-support properly
```

Comment:
Does kube-state-metrics indicate the presence of services that are in a configuration that isn't working?  It seems like a good use-case for determine if load balancer services are working as users expect.

Thinking about the SLI question in the PRR below.

Reply:
I am not familiar with kube-state-metrics. I have just read what it can show about services, and my understanding is, that the "isn't working" state can be obtained on an indirect way only: there is no LB IP/Hostname for the service if it does not work https://github.com/kubernetes/kube-state-metrics/blob/12402a564cbf4557763079ab8e6e995d9afb4db9/docs/service-metrics.md

Reply:
If lack of SLI is a concern, we can also have a dedicated event type for when this error occurs?

Reply:
> I am not familiar with kube-state-metrics. I have just read what it can show about services, and my understanding is, that the "isn't working" state can be obtained on an indirect way only: there is no LB IP/Hostname for the service if it does not work https://github.com/kubernetes/kube-state-metrics/blob/12402a564cbf4557763079ab8e6e995d9afb4db9/docs/service-metrics.md

Can you update the PRR referencing the kube-state-metrics and how to interpret them.  If a cluster-admin chooses to enable kube-state-metrics is up to them, but describing to them how to determine if there is a systemic problem at scale is important.

Reply:
I updated the KEP with the kube-state-metrics reference.